content/issues/organizing-your-work-with-project-boards/managing-project-boards/changing-project-board-visibility.md

```diff
@@ -24,6 +24,8 @@ shortTitle: Change visibility
 
 {% endtip %}
 
+Only organization members with *owner* or *project admin* roles can change project board visibility.
```

Comment:
Thanks @riteshsp2000! Let's tweak the phrasing as below, and also move it to a `permissions:` front matter:

- https://github.com/github/docs/tree/main/content#permissions
- https://github.com/github/docs/blob/main/contributing/content-model.md#permissions-statements

>Organization owners and people with admin access for a project board can change a project board's visibility.

Reply:
#